scripts/ci/in_container/kubernetes/app/deploy_app.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -eu
+set -x
+MY_DIR=$(cd "$(dirname "$0")" && pwd)
+
+AIRFLOW_SOURCES=$(
+    cd "${MY_DIR}/../../../../../" || exit 1
+    pwd
+)
+export AIRFLOW_SOURCES
+
+# shellcheck source=scripts/ci/in_container/_in_container_utils.sh
+. "${MY_DIR}/../../_in_container_utils.sh"
+
+assert_in_container
+
+in_container_script_start
+
+export TEMPLATE_DIRNAME="${MY_DIR}/templates"
+export BUILD_DIRNAME="${MY_DIR}/build"
+
+# shellcheck source=common/_autodetect_variables.sh
+. "${AIRFLOW_SOURCES}/common/_autodetect_variables.sh"
+
+# Source branch will be set in DockerHub
+SOURCE_BRANCH=${SOURCE_BRANCH:=${DEFAULT_BRANCH}}
+# if AIRFLOW_CONTAINER_BRANCH_NAME is not set it will be set to either SOURCE_BRANCH (if overridden)
+# or default branch for the sources
+AIRFLOW_CONTAINER_BRANCH_NAME=${AIRFLOW_CONTAINER_BRANCH_NAME:=${SOURCE_BRANCH}}
+
+if [[ ! -d "${BUILD_DIRNAME}" ]]; then
+    mkdir -p "${BUILD_DIRNAME}"
+fi
+
+rm -f "${BUILD_DIRNAME}"/*
+
+if [[ "${KUBERNETES_MODE}" == "persistent_mode" ]]; then
+    INIT_DAGS_VOLUME_NAME=airflow-dags
+    POD_AIRFLOW_DAGS_VOLUME_NAME=airflow-dags
+    CONFIGMAP_DAGS_FOLDER=/root/airflow/dags
+    CONFIGMAP_GIT_DAGS_FOLDER_MOUNT_POINT=
+    CONFIGMAP_DAGS_VOLUME_CLAIM=airflow-dags
+else
+    INIT_DAGS_VOLUME_NAME=airflow-dags-fake
+    POD_AIRFLOW_DAGS_VOLUME_NAME=airflow-dags-git
+    CONFIGMAP_DAGS_FOLDER=/root/airflow/dags/repo/airflow/contrib/example_dags
+    CONFIGMAP_GIT_DAGS_FOLDER_MOUNT_POINT=/root/airflow/dags
+    CONFIGMAP_DAGS_VOLUME_CLAIM=
+fi
+
+CONFIGMAP_GIT_REPO=${TRAVIS_REPO_SLUG:-apache/airflow}
+CONFIGMAP_BRANCH=${DEFAULT_BRANCH:=master}
+
+SED_COMMAND="sed"
```

Comment:
What is the purpose of this variable?

Reply:
That was left-over after I removed Darwin case (it was both OSX and Linux but now it is in container so only linux is needed. removed.